static/app/views/replays/detail/network/index.tsx

```diff
@@ -260,11 +215,11 @@ function NetworkList({replayRecord, networkSpans}: Props) {
             {({width, height}) => (
               <MultiGrid
                 ref={multiGridRef}
-                columnCount={columns.length}
+                columnCount={COLUMN_COUNT}
                 columnWidth={({index}) => {
                   if (index === 1) {
                     return Math.max(
-                      columns.reduce(
+                      range(COLUMN_COUNT).reduce(
```

Comment:
We could replace `lodash/range` with

```suggestion
                      Array.from(Array(COLUMN_COUNT)).reduce(
```